src/ipyradiant/basic_tools/custom_uri_ref.py

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2020 ipyradiant contributors.
+# Distributed under the terms of the Modified BSD License.
+from typing import Union
+
+from rdflib import Graph, URIRef
+from rdflib.namespace import NamespaceManager
+
+
+class CustomURIRef:
```

Comment:
I think we may want to use this as the key for nodes in the collapsed graph so that they show up nice and pretty as well.